molgenis/molgenis-emx2
# Fix/3605 sort repeats numeric

What are the main changes you did:
- added sort option to repeat fetch 

how to test:
- inspect the repeated vars and see var_2 comes before var_10

Closes #3605

todo:
